File: MODUL 6/SOAL 3/PRAK603-2410817210026-FadhilSyahdamaMahatmaPutra.py
pozem, changli = map(int, input().split())

if pozem != changli:
    print("Jumlah tidak sama")
    exit()

x = list(map(int, input().split()))
y = list(map(int, input().split()))

for i in range(pozem):
    print(x[i] * y[i], end=" ")

File: MODUL 6/SOAL 3/PRAK603-2410817210026-FadhilSyahdamaMahatmaPutra.c
#include <stdio.h>
int main(){
    int pozem, changli, i, j;

    scanf("%d %d", &pozem, &changli);

    if(pozem != changli){
        printf("Jumlah tidak sama");
        return 1;
    }

    int x[pozem], y[changli];

    for(i = 0; i < pozem; i++){
        scanf("%d", &x[i]);
    }

    for(j = 0; j < changli; j++){
        scanf("%d", &y[j]);
    }

    for(i = 0; i < pozem; i++){
        printf("%d ", x[i] * y[i]);
    }
}  